debian/base/Dockerfile

```diff
@@ -20,10 +20,11 @@ LABEL io.confluent.docker.git.id=$COMMIT_ID
 ARG BUILD_NUMBER=-1
 LABEL io.confluent.docker.build.number=$BUILD_NUMBER
 
-ARG CONFLUENT_DEB_REPO="http://packages.confluent.io"
-ARG APT_ALLOW_UNAUTHENTICATED=false
+ARG CONFLUENT_PACKAGES_REPO=$CONFLUENT_PACKAGES_REPO
+
+ARG ALLOW_UNSIGNED=false
 #Set an env var so that it's available in derived images
-ENV APT_ALLOW_UNAUTHENTICATED=${APT_ALLOW_UNAUTHENTICATED}
+ENV ALLOW_UNSIGNED=$ALLOW_UNSIGNED
```

Comment:
Sure, and we might say this case is fine since these should be built off of our signed repos for release.

That said, I'm not sure what docs you're referring to. We have some examples in that code, but there are plenty more env variables in each of the dockerfiles derived from this one that people rely on and are not explicitly documented afaict.